.storybook/webpack.config.js

```diff
@@ -78,7 +78,7 @@ module.exports = (baseConfig, env) => {
       stats: "errors-only",
     },
     resolve: {
-      extensions: [".js", ".jsx", ".ts", ".tsx"],
+      extensions: [".mjs", ".js", ".jsx", ".ts", ".tsx"],
```

Comment:
Do we use `mjs`?

Reply:
No, but `react-relay-network-modern` packages / publishes `.mjs` files and webpack will complain. I've increasingly started seeing other libraries do the same (I believe `graphql-js`, for example).

Reply:
I see 👍